app/scripts/modules/core/src/pipeline/executions/execution/executionMarker.less

```diff
@@ -42,3 +42,11 @@
     position: relative;
   }
 }
+
+.execution-marker-waiting {
+  color: var(--color-black);
+  background-color: var(--color-alert) !important;
+  .fa {
```

Comment:
Let's use a more detailed classname here. `fa` is a generic class for our icons, and I wouldn't want this to unintentionally affect other icons.

Reply:
Since .fa is already scoped inside .execution-marker-waiting, So it will not affect any other icons.

Reply:
I agree  with Casey, let's change this from `fa` as that classname already has a very specific meaning in Deck and is globally scoped

Reply:
As suggested we have added custom classes to increase the styles specificity.